.travis.yml

```
@@ -52,8 +40,6 @@ matrix:
 before_install:
   - phpenv config-rm xdebug.ini || true
   - composer self-update
```

```suggestion
  - composer self-update --2
```
i suggest we start using composer 2. that should make the installations much faster according to reports of people who tried it.